modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Filters.tsx

```diff
@@ -100,7 +100,7 @@ function AddFDSFilterModalContent({
 		boolean
 	>();
 	const [from, setFrom] = useState<string>(
-		(filter as IDateFilter)?.from ?? format(new Date(), 'yyyy-MM-dd')
+		(filter as IDateFilter)?.from ?? ''
```

Comment:
Any reason to remove the defaults? I don't mind the blank default, just curious.